package.json

```diff
@@ -36,7 +40,7 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^5.3.0",
     "devtron": "^1.3.0",
-    "electron-builder": "5.2.1",
+    "electron-builder": "7.9.0",
```

Comment:
v7.9.0 has the bug, generated app can't start. You should use [v7.10.2](https://github.com/electron-userland/electron-builder/releases/tag/v7.10.2).